log/sequencer.go

```diff
@@ -166,8 +171,12 @@ func (s Sequencer) createRootSignature(ctx context.Context, root trillian.Signed
 // which will fail if the tx was committed. Should only do this if we can hide the details of
 // the underlying storage transactions and it doesn't create other problems.
 func (s Sequencer) SequenceBatch(ctx context.Context, limit int) (int, error) {
-	tx, err := s.logStorage.Begin(ctx)
+	logID, err := util.LogID(ctx)
```

Comment:
(In a second PR), consider removing logID from ctx.  This is an unexpected place to store such an important variable. It hides an input variable.  Calling this with the expected context.Background() will fail.